CONFIGURATION.md

```diff
@@ -69,6 +69,12 @@ tests:
     target_cloud: ''
   openshift_ansible_src:
     target_cloud: ''
+  openshift_ansible_upgrade:
+    target_cloud: ''
+    previous_ansible_version: ''
```

Comment:
This seems like it could mirror `tag_specification` and fully define the previous release, so you don't need to explicitly provide the version, image and RPMs